org.lflang/src/org/lflang/TargetConfig.java

```diff
@@ -129,159 +131,161 @@
+         * FIXME
+         * The default is 10.
+         */
+        public int trials = 10;
+
+        /**
+         * Used to create an artificial clock synchronization error for the purpose of testing.
+         * The default is null.
+         */
+        public TimeValue testOffset;
+    }
 
-/**
- * Settings related to coordination of federated execution.
- */
-class CoordinationOptions {
-    
     /**
-     * For centralized coordination, if a federate has a physical action that can trigger
-     * an output, directly or indirectly, then it will send NET (next event tag) messages
-     * to the RTI periodically as its physical clock advances. This option sets the amount
-     * of time to wait between sending such messages. Increasing this value results in
-     * downstream federates that lag further behind physical time (if the "after" delays
-     * are insufficient).
-     * The default is null, which means it is up the implementation to choose an interval.
+     * Settings related to coordination of federated execution.
      */
-    public TimeValue advance_message_interval = null;
-}
+    public static class CoordinationOptions {
+
+        /**
+         * For centralized coordination, if a federate has a physical action that can trigger
+         * an output, directly or indirectly, then it will send NET (next event tag) messages
+         * to the RTI periodically as its physical clock advances. This option sets the amount
+         * of time to wait between sending such messages. Increasing this value results in
+         * downstream federates that lag further behind physical time (if the "after" delays
+         * are insufficient).
+         * The default is null, which means it is up the implementation to choose an interval.
+         */
+        public TimeValue advance_message_interval = null;
+    }
 
-/**
- * Settings related to Docker options.
- */
-class DockerOptions {
     /**
-     * The base image and tag from which to build the Docker image. The default is "alpine:latest".
+     * Settings related to Docker options.
      */
-    public String from = "alpine:latest"
-}
+    public static class DockerOptions {
+        /**
+         * The base image and tag from which to build the Docker image. The default is "alpine:latest".
+         */
+        public String from = "alpine:latest";
+    }
 
-enum Mode {
-    STANDALONE,
-    INTEGRATED,
-    UNDEFINED
-}
+    public enum Mode {
+        STANDALONE,
+        INTEGRATED,
+        UNDEFINED
+    }
 
-/**
- * Settings related to tracing options.
- */
-class TracingOptions {
     /**
-     * The name to use as the root of the trace file produced.
-     * This defaults to the name of the .lf file.
+     * Settings related to tracing options.
      */
-    public String traceFileName = null
+    public static class TracingOptions {
+        /**
+         * The name to use as the root of the trace file produced.
+         * This defaults to the name of the .lf file.
```

Comment:
This comment does not describe the code below, so the default is probably set elsewhere, but that makes it a little confusing...